BUILD

```diff
@@ -4208,19 +4208,62 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
+    name = "ssl_transport_security_utils",
+    srcs = [
+        "//src/core:tsi/ssl/key_logging/ssl_key_logging.cc",
+        "//src/core:tsi/ssl_transport_security_utils.cc",
+    ],
+    hdrs = [
+        "//src/core:tsi/ssl/key_logging/ssl_key_logging.h",
+        "//src/core:tsi/ssl_transport_security_utils.h",
+    ],
+    external_deps = [
+        "absl/base:core_headers",
+        "absl/log:check",
+        "absl/log:log",
+        "absl/status",
+        "absl/status:statusor",
+        "absl/strings",
+        "libcrypto",
+        "libssl",
+    ],
+    visibility = ["//visibility:public"],
```

Comment:
Do we really want this to be publicly-visible? Can we scope down the visibility to either internal-only or (even better) only those directories that need this visibility?

Reply:
IIRC this doesn't make it a public header (those are the grpc public header targets) - for example a few lines up, the `tsi_alts_frame_protector` is set to public visibility?

We could also scope the visibility to specific packages, I just don't want to take the time to do that if it doesn't seem it's a mechanism grpc really uses

Reply:
What visibility level are we looking for then - a very fine grained of everywhere this is used, or would you suggest something broader?